backend/core/domains/payments/services/payment_event_sourcing_service.py

```
@@ -0,0 +1,565 @@
+            'status': 'CREATED',
+            'amount': str(payment.amount),
+            'currency': payment.currency,
+            'created_at': payment.created_at.isoformat()
+        }
+
+        replay_log = []
+        current_state = initial_state.copy()
+        events_processed = 0
+
+        for event in events:
+            event_data = event.event_data
```

Ruff (0.13.3)

Local variable `event_data` is assigned to but never used

Remove assignment to unused variable `event_data`

(F841)